src/index.ts

```diff
@@ -3,7 +3,7 @@
  * but is also a useful utility for operating on GraphQL files and building
  * sophisticated tools.
  *
- * This primary module exports a general purpose function for fulfilling all
+ * Ths primary module exports a general purpose function for fulfilling all
```

Comment:
Hi @IvanGoncharov you missed `i` 😅

Reply:
Yes, exectly.
this is a change to test how well this script works in CI pipeline